components/ui/navigation/window-controls.tsx

```
@@ -11,55 +11,65 @@ interface WindowControlsProps {
       </button>
       {/* Minimize Button */}
       <button
         aria-label="Minimize"
         title="Minimize"
-        className="relative group w-3 h-3 rounded-full bg-yellow-500 hover:bg-yellow-600 transition-colors flex items-center justify-center"
+        className={`relative group ${buttonSize} rounded-full bg-yellow-500 hover:bg-yellow-600 transition-colors flex items-center justify-center`}
         onClick={onMinimize}
         disabled={!onMinimize} // Disable based solely on prop presence
       >
-        <HoverIcon icon="−" />
+        <HoverIcon icon="−" size={size} />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)